en/middleware/micrortps.md

```diff
@@ -70,7 +70,7 @@ This is needed because the first version of ROS does not support RTPS.
 
 ## Code generation
 
-> **Note** [Fast RTPS must be installed](../setup/fast-rtps-installation.md) in order to generate the required code!
+> **Note** [Fast RTPS 1.8.2 or later must be installed](../setup/fast-rtps-installation.md) in order to generate the required code!
 > *Fast RTPS* is installed *by default* if you use the normal installers/scripts for [macOS](../setup/dev_env_mac.md), [Windows Cygwin](../setup/dev_env_windows_cygwin.md) or [Ubuntu](../setup/dev_env_linux_ubuntu.md).
```

Comment:
@TSC21 we should remove Ubuntu from this: "Fast RTPS is installed by default if you use the normal installers/scripts for macOS, Windows Cygwin or Ubuntu."

Reply:
Yes we should. I will create a build script for the ROS2 development environment after and we can add it here.

Reply:
OK. I will update this one.